package.json

```diff
@@ -18,10 +18,7 @@
     "build": "babel src -d build --ignore '**/*.test.js' --ignore '**/__mocks__/*' && cp src/*.json build",
     "start": "node build/server.js",
     "start:local": "npm run db:migrate ; babel-node -r dotenv/config src/server.js",
```

Comment:
Doesn't this line `npm run db:migrate` need to be removed?

Reply:
Sure! Just removed this